README.md

```diff
@@ -98,39 +98,39 @@ The expected result should be:
 
 ### TASK #2 - React & Hooks
 
-Implement `<RocketsList>` component with the following specifications:
+Implement the `<RocketsList>` component with the following specifications:
 
-- The component receives `filerParams` object (with the shape described in [task #1][task-1]) as a prop.
-- The component obtains a list of 'missions' from a [custom hook][custom-hook], which fetches **the whole list of missions** from the [SpaceX API][spacex-api] and processes them with `prepareData` function (from [task #1][task-1]) and `filterParams` prop.
+- The component receives a `filterParams` object (with the shape described in [task #1][task-1]) as a prop.
+- The component obtains a list of 'missions' from a [custom hook][custom-hook], which fetches **the entire list of missions** from the [SpaceX API][spacex-api] and processes them using the `prepareData` function (from [task #1][task-1]) and the `filterParams` prop.
   - The mission data should only be fetched once.
-  - As a part of this challenge, you are not allowed to use any of the filter parameters provided by the [SpaceX API docs][spacex-api-docs].
-- For each 'mission' obtained from the custom hook, the component renders a string using [template literals][template-literals] with the following format: "#`flight_number` `mission_name` (`payloads_count`)"
+  - As part of this challenge, you are not allowed to use any of the filter parameters provided by the [SpaceX API docs][spacex-api-docs].
+- For each 'mission' obtained from the custom hook, the component renders a string using [template literals][template-literals] in the following format: "#`flight_number` `mission_name` (`payloads_count`)".
 - While 'missions' are being fetched from the API, the component renders `"Loading..."` on the screen.
 - If no 'missions' are obtained from the custom hook, the component renders `"No data"` on the screen.
 
 **Expected outcome:**
 
-Please make sure to set the `filterParams` to match the example below:
+Please set the `filterParams` to match the example below:
 
 ```js
-<RocketList
+<RocketsList
```

Comment:
```suggestion
<RocketList
```
we aren't updating it, right?

Reply:
In the title of the task we ask to `Implement the `<RocketsList>` component`, so this change makes sense.